packages/dropdowns.next/src/elements/menu/utils.ts

```diff
@@ -33,45 +50,76 @@ type FloatingPlacement = (typeof FLOATING_PLACEMENT)[number];
+): FloatingPlacement => {
   let retVal;
 
-  if (placement !== undefined) {
-    /* Map Garden to Floating-UI placements */
-    const placementMap: Record<string, FloatingPlacement> = {
-      end: 'right',
-      'end-top': 'right-start',
-      'end-bottom': 'right-end',
-      start: 'left',
-      'start-top': 'left-start',
-      'start-bottom': 'left-end'
+  /* Map Garden to Floating-UI placements */
+  const placementMap: Record<string, FloatingPlacement> = {
+    auto: 'bottom-start',
+    end: 'right',
+    'end-top': 'right-start',
+    'end-bottom': 'right-end',
+    start: 'left',
+    'start-top': 'left-start',
+    'start-bottom': 'left-end'
+  };
+
+  retVal = placementMap[placement] || placement;
+
+  if (isRtl) {
+    /* Map Floating-UI to RTL placements */
+    const placementMapRtl: Record<string, FloatingPlacement> = {
+      left: 'right',
+      'left-start': 'right-start',
+      'left-end': 'right-end',
+      'top-start': 'top-end',
+      'top-end': 'top-start',
+      right: 'left',
+      'right-start': 'left-start',
+      'right-end': 'left-end',
+      'bottom-start': 'bottom-end',
+      'bottom-end': 'bottom-start'
     };
 
-    retVal = placementMap[placement] || placement;
-
-    if (isRtl) {
-      /* Map Floating-UI to RTL placements */
-      const placementMapRtl: Record<string, FloatingPlacement> = {
-        left: 'right',
-        'left-start': 'right-start',
-        'left-end': 'right-end',
-        'top-start': 'top-end',
-        'top-end': 'top-start',
-        right: 'left',
-        'right-start': 'left-start',
-        'right-end': 'left-end',
-        'bottom-start': 'bottom-end',
-        'bottom-end': 'bottom-start'
-      };
-
-      retVal = placementMapRtl[retVal] || retVal;
-    }
+    retVal = placementMapRtl[retVal] || retVal;
   }
 
   return retVal;
 };
 
+/**
+ * If `fallbackPlacements` are given, converts them to FLoating-UI placements.
+ *
+ * Otherwise, builds a list of Floating-UI fallback placements given the
+ * current `placement`.
+ *
+ * @param isRtl Determines if layout is right-to-left.
+ * @param placement A Floating-UI placement.
+ * @param fallbackPlacements Optional list of Garden placements
+ *
+ * @returns A list of Floating-UI fallback placements.
+ */
+export const getFallbackPlacements = (
+  isRtl: boolean,
+  placement: FloatingPlacement,
+  fallbackPlacements?: GardenPlacement[]
+): Placement[] => {
+  if (Array.isArray(fallbackPlacements) && fallbackPlacements.length > 0) {
+    return fallbackPlacements.map(fallback => toFloatingPlacement(isRtl, fallback) as Placement);
+  }
+
+  const side = placement.split('-')[0];
+  const sameSideFallbacks = [...Placements[side]];
+  const oppositeSideFallbacks = Placements[oppositeSideMap[side]];
+
+  // Remove the primary placement from the list of same-side fallbacks
+  // to prevent extra work for Floating-UI
+  sameSideFallbacks.splice(sameSideFallbacks.indexOf(placement as Placement), 1);
```

Comment:
There were some oddities with including crossAxis fallback placements, so by default, `Menu` won't offer that (this is in parity with the legacy Menu). If cross-axis becomes a necessary feature for consumers, the use of Garden-flavor placement values in `fallbackPlacements` is a way to achieve that behavior.